File: filesets/language/PHP/UntitledTest.php
<?php

require_once 'PHPUnit/Framework.php';
require_once 'Untitled.php';

class UntitledTest extends PHPUnit_Framework_TestCase
{
    public function testAnswer()
    {
        $this->assertEquals(9 * 6, answer());
    }
}

?>

File: languages/Java-1.8_Mockito/UntitledTest.java
import org.junit.*;
import static org.junit.Assert.*;
import static org.mockito.Mockito.*;

public class UntitledTest {

    @Test
    public void hitch_hiker() {
        UntitledHelper mock = mock(UntitledHelper.class);
        Untitled target = new Untitled(mock);
        int expected = 42;
        when(mock.answer()).thenReturn(6 * 9);
        int actual = target.answer();
        assertEquals(expected, actual);
    }
}
